src/cmd/create.go

```diff
@@ -338,7 +338,6 @@ func createContainer(container, image, release string, showCommandToEnter bool)
 		"create",
 		"--dns", "none",
 		"--env", toolboxPathEnvArg,
-		"--group-add", sudoGroup,
```

Comment:
Do we really not need the group for `sudo`?

Reply:
Well, in the case of Fedora-based images it works but other distros may use `wheel` as the sudoers group and this can't be tested on the host but only inside of the container.

Reply:
For those following along at home, this was because of a [regression](https://github.com/containers/libpod/pull/6715) in Podman where the `--group-add` and `--privileged` flags won't work together.